node:assert:*
      throw err;
      ^

AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
+ actual - expected

+ Comparison {}
- Comparison {
-   bar: true
- }
    at Object.<anonymous> (*assert_throws_stack.js:*:*)
    at Module._compile (node:internal*modules*cjs*loader:*:*)
    at Module._extensions..js (node:internal*modules*cjs*loader:*:*)
    at Module.load (node:internal*modules*cjs*loader:*:*)
    at Module._load (node:internal*modules*cjs*loader:*:*)
    at Function.executeUserEntryPoint [as runMain] (node:internal*modules*run_main:*:*)
    at node:internal*main*run_main_module:*:* {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: Error: foo
      at assert.throws.bar (*assert_throws_stack.js:*:*)
      at getActual (node:assert:*:*)
      at Function.throws (node:assert:*:*)
      at Object.<anonymous> (*assert_throws_stack.js:*:*)
      at Module._compile (node:internal*modules*cjs*loader:*:*)
      at Module._extensions..js (node:internal*modules*cjs*loader:*:*)
      at Module.load (node:internal*modules*cjs*loader:*:*)
      at Module._load (node:internal*modules*cjs*loader:*:*)
      at Function.executeUserEntryPoint [as runMain] (node:internal*modules*run_main:*:*)
      at node:internal*main*run_main_module:*:*,
  expected: { bar: true },
  operator: 'throws'
}

Node.js *
